src/api/json/catalog.json

```diff
@@ -2010,6 +2010,42 @@
       "fileMatch": ["admin/config*.yml"],
       "url": "https://json.schemastore.org/netlify.json"
     },
+    {
+      "name": "New Horizons Planet",
+      "description": "Describes a planet config for the New Horizons framework",
+      "fileMatch": [
+        "planets/**.json",
+        "planets/**.jsonc"
+      ],
+      "url": "https://raw.githubusercontent.com/Outer-Wilds-New-Horizons/new-horizons/main/NewHorizons/Schemas/body_schema.json"
+    },
+    {
+      "name": "New Horizons System",
+      "description": "Describes a star system config for the New Horizons framework",
+      "fileMatch": [
+        "systems/**.json",
+        "systems/**.jsonc"
+      ],
+      "url": "https://raw.githubusercontent.com/Outer-Wilds-New-Horizons/new-horizons/main/NewHorizons/Schemas/system_schema.json"
+    },
+    {
+      "name": "New Horizons Translation",
+      "description": "Describes a translation config for the New Horizons framework",
+      "fileMatch": [
```

Comment:
These file match patterns seem a little too generic and I suspect they will create a lot of false positives. Can you come up with a pattern that is less generic?